### PR TITLE
Bump to citus 10.1.4

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: citus.spec [hunk 1]
```diff
@@ -43,8 +43,6 @@ make %{?_smp_mflags}
 # Install documentation with a better name:
 %{__mkdir} -p %{buildroot}%{pginstdir}/doc/extension
 %{__cp} README.md %{buildroot}%{pginstdir}/doc/extension/README-%{sname}.md
-%{__cp} NOTICE %{buildroot}%{pginstdir}/doc/extension/NOTICE-%{sname}
```
Comment (hanefi) [inline citus.spec]: Why did we remove this NOTICE file here?

File: pkgvars [hunk 2]
```diff
@@ -1,4 +1,6 @@
 pkgname=citus
 pkgdesc='Citus (Open-Source)'
-pkglatest=10.2.3.citus-1
+pkglatest=10.1.4.citus-1
+releasepg=11,12,13
+nightlypg=12,13
```
Comment (hanefi) [inline pkgvars]: I do not think these fields make sense.

1. Do we need `releasepg` here?
2. Do we need `nightlypg` here?
3. PG11 support was dropped in `10.1` and we should not have that in this field here.

File: citus.spec [hunk 3]
```diff
@@ -44,7 +44,6 @@ make %{?_smp_mflags}
 %{__mkdir} -p %{buildroot}%{pginstdir}/doc/extension
 %{__cp} README.md %{buildroot}%{pginstdir}/doc/extension/README-%{sname}.md
 %{__cp} NOTICE %{buildroot}%{pginstdir}/doc/extension/NOTICE-%{sname}
-
```
Comment (hanefi) [inline citus.spec]: Let's keep the empty line that makes the file easier to read.

File: citus.spec (hunk 1 above)
Comment (gurkanindibay) [reply citus.spec]: Thanks for the catch. We have an open issue about updating templates. I need to update it manually until implementing this issue. I missed the manual check here

File: pkgvars (hunk 2 above)
Comment (gurkanindibay) [reply pkgvars]: I missed that part as well. I deleted this part as well. Thanks for the comment

File: citus.spec (hunk 3 above)
Comment (gurkanindibay) [reply citus.spec]: Fixed as you suggested

**Review by hanefi — APPROVED** [review]
Please go over the test outputs before merging this PR

**Pull request merged**